src/app/dev/DevToys.Blazor/Components/Text/MonacoEditor/MonacoEditor.razor.cs

```diff
@@ -77,13 +77,13 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
             options.MouseWheelZoom = false;
             options.OverviewRulerBorder = false;
             options.ScrollBeyondLastLine = false;
-            options.FontLigatures = false;
+            options.FontLigatures = true;
```

Comment:
Unrelated change, but nice to have. It enabled font ligatures as it says. Depending on user feedback, we could consider adding an option in `Settings` to enable/disable ligatures.